partner_ref_sequence/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module will introduce automatic integer number generation for customer.
+And fill in ref field
```

Comment:
`This module provides a sequence to generate an automatic number to set the customer number.`